ee/vellum_ee/workflows/display/nodes/vellum/base_node.py

```diff
@@ -62,6 +64,24 @@ def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> J
                     }
                 )
 
+        outputs: JsonArray = []
+        for output in node.Outputs:
+            type = primitive_type_to_vellum_variable_type(output)
+            value = (
+                self.serialize_value(display_context, output.instance)
+                if output.instance is not None and output.instance != UNDEF
```

Comment:
I could see us introducing an `is_nil` helper to do this check as I suspect we'll be doing this a lot.

```suggestion
                if output.instance is not None and output.instance != UNDEF
```

Reply:
I agree it's a good helper to have. Surprisingly though, this is only the second place in the code where we do a `==` or `!=` check against UNDEF :O